doc/documentation_data_format.md

```diff
@@ -229,10 +181,10 @@ numeric value or `inf` (lower-case) for steady-state measurements.
 - `observableParameters` [STRING OR NULL, OPTIONAL]
 
   This field allows overriding or introducing condition-specific versions of
-  parameters defined in the model. The model can define observables (see above)
-  containing place-holder parameters which can be replaced by
-  condition-specific dynamic or constant parameters. Placeholder parameters
-  must be named `observableParameter${n}_${observableId}`
+  output parameters defined in the observation model. The model can define
+  observables (see below) containing place-holder parameters which can be
+  replaced by condition-specific dynamic or constant parameters. Placeholder
+  parameters must be named `observableParameter${n}_${observableId}`
```

Comment:
do we want to keep that for the moment? or use the a=1;b=2 thing?

Reply:
I think decision is still pending. All changes still follow this format.